index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Style Guide</title>
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <style>
+        @font-face {
+            font-family: SpectralReg;
+            src: url("fonts/Spectral/Spectral-Light.ttf");
+        }
+        @font-face {
+            font-family: SpectralItalic;
+            src: url("fonts/Spectral/Spectral-LightItalic.ttf");
+        }
+        @font-face {
+            font-family: SpectralBold;
+            src: url("fonts/Spectral/Spectral-Bold.ttf");
+        }
+        @font-face {
+            font-family: SansReg;
+            src: url("fonts/Source_Sans_Pro/SourceSansPro-Regular.ttf");
+        }
+        @font-face {
+            font-family: SansItalic;
+            src: url("fonts/Source_Sans_Pro/SourceSansPro-Italic.ttf");
+        }
+        @font-face {
+            font-family: SansBold;
+            src: url("fonts/Source_Sans_Pro/SourceSansPro-Bold.ttf");
+        }
+        @font-face {
+            font-family: SansSemiBold;
+            src: url("fonts/Source_Sans_Pro/SourceSansPro-SemiBold.ttf");
+        }
+        @font-face {
+            font-family: SpectralMedium;
+            src: url("fonts/Spectral/Spectral-Medium.ttf");
+        }
+        @font-face {
```

Comment:
nit: in a production setting, would probably want to only include fonts (or any assets) actually being used.